<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>615</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777214</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>770</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>620</width>
      <height>400</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Welcome to InterCloud RAIDER:</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>351</width>
       <height>40</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>To start working with InterCloudRAIDER, select</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>251</width>
       <height>40</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>For help getting started, select</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>521</y>
      <width>107</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>521</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="CancelOperationButton">
       <property name="text">
        <string>   Cancel Operation  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StartNewOperationButton">
       <property name="text">
        <string>Start New Operation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionView_Operation_Log"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPDP_Verification"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionParameter_Setting"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuSetup"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionParameter_Setting">
   <property name="text">
    <string>Background Parameter</string>
   </property>
  </action>
  <action name="actionView_Operation_Log">
   <property name="text">
    <string>View Operation Log</string>
   </property>
  </action>
  <action name="actionPDP_Verification">
   <property name="text">
    <string>PDP Verification</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>StartNewOperationButton</tabstop>
  <tabstop>CancelOperationButton</tabstop>
  <tabstop>RunButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
